modules/model/shared/src/main/scala/observe/model/SequenceState.scala

```diff
@@ -17,24 +17,25 @@ import monocle.macros.GenPrism
 enum SequenceState(val name: String) derives Eq, Encoder, Decoder:
   case Idle                extends SequenceState("Idle")
   case Running(
-    userStop:            Boolean,
-    internalStop:        Boolean,
-    isWaitingUserPrompt: Boolean,
-    isStarting:          Boolean
+    userStop:          Boolean,
+    internalStop:      Boolean,
+    waitingUserPrompt: Boolean,
+    waitingNextAtom:   Boolean,
+    starting:          Boolean
```

Comment:
Is this true only once?

Reply:
Whenever `start` is invoked in `ObserveEngine` it's set to true.